synapse/handlers/ui_auth/__init__.py

```diff
@@ -38,4 +38,4 @@ class UIAuthSessionDataConstants:
     # used during registration to store the registration token used (if required) so that:
     # - we can prevent a token being used twice by one session
     # - we can 'use up' the token after registration has successfully completed
-    REGISTRATION_TOKEN = "org.matrix.msc3231.login.registration_token"
+    REGISTRATION_TOKEN = "m.login.registration_token"
```

Comment:
We should note that any pending UIA sessions that existing before and after the change will no longer have their registration token step completed. However, I believe this is such a small edge-case that we can safely ignore it.

Reply:
Maybe this'll also be mentioned in the `upgrade.md` docs?

Reply:
Since users can just attempt to re-register to get around it, I don't think it's really that noteworthy.